pkgs/by-name/ni/nixos-rebuild-ng/src/nixos_rebuild/nix.py

```diff
@@ -433,14 +433,14 @@ def get_generation_info(generation: Generation) -> GenerationJson:
         )
         try:
             nixos_version = (generation_path / "nixos-version").read_text().strip()
-        except IOError as ex:
+        except OSError as ex:
```

Comment:
Technically what we want here is an `FileNotFoundError` instead because this is the only case that I can think that this can genuinely happen. But at the same time not sure how much this is worth it changing, considering that we can still debug the error if we look at the stack trace from the log if we want.